src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -580,7 +580,7 @@ public static short CONST_SHORT(final int v) {
      * @param object  the {@link Object} to test, may be {@code null}
      * @param defaultValue  the default value to return, may be {@code null}
      * @return {@code object} if it is not {@code null}, defaultValue otherwise
-     * TODO Rename to getIfNull in 4.0
```

Comment:
@brettkail-wk 

kindly consider TODO.

Reply:
> enough better

not saying anything. Just assuming rename implies a breaking change.

Not consider it to be happen anyways.

Thanks for the interaction.